packages/databyss-ui/components/Editor/Menu/FormatMenu.js

```diff
@@ -0,0 +1,218 @@
+import React, { useEffect, useRef } from 'react'
+import { Button, Text, HoverView } from '@databyss-org/ui/primitives'
+import { isMobileOs } from '@databyss-org/ui/'
+import space from '@databyss-org/ui/theming/space'
+import { useEditorContext } from '../EditorProvider'
+import { toggleMark, startTag } from '../state/actions'
+
+const mobileActions = [
+  {
+    type: 'SOURCE',
+    label: '+ source',
+    variant: 'uiTextNormal',
+    action: a => startTag(a),
+  },
+  {
+    type: 'TOPIC',
+    label: '+ topic',
+    variant: 'uiTextNormal',
+    action: a => startTag(a),
+  },
+  {
+    type: 'LOCATION',
+    label: '+ location',
+    variant: 'uiTextNormal',
+    action: a => startTag(a),
+  },
+]
+
+const webActions = [
```

Comment:
I'm using the convention `desktop...` for things like this, because `mobile` is web as well. So the three prefixes are `mobile__`, `desktop__` and `native__`